float32 transl_error
float32 rot_error

string name
